less/components/profile.less

```diff
@@ -0,0 +1,69 @@
+@import "../variables";
+
+.profile-page {
+
+    &&&& label {
+        color: #777777;
+
+        input[type="checkbox"] {
+            margin-left: @spacer;
+        }
+    }
+
+    p.success {
+        margin: @spacer/2;
+        color: #42a94b;
+        font-weight: 500;
+    }
+
+    p.failure {
+        margin: @spacer/2;
+        color: #a94442;
+        font-weight: 500;
+    }
+
+    .flex-row {
+        display: flex;
+        align-items: baseline;
+    }
+
+    .input-container {
+        max-width: 300px;
+    }
+
+    .settings-group {
+        background-color: white;
+        padding: @spacer 4*@spacer;
+        margin: 2*@spacer 0;
+        border: 1px solid #dddddd;
+        position: relative;
+    }
+
+    .settings-loader {
```

Comment:
Consider either re-using a loader from https://github.com/intermine/bluegenes/blob/dev/less/components/loader.less, or adding this loader to the loaders file - we should try to re-use loaders so we get a consistent look and feel between different pages.  

Also, what's the best way to trigger this loader in the UI? I wasn't sure how to see it.

Reply:
It does actually use the `mini-loader` component. This CSS is just to fade the panel when the loader is present. I can consider generalizing it if I find I need the same fading and centering anywhere else, but right now it's only used for the user preferences panel.

You can trigger it in Chrome by opening Bluegenes' home page, setting network to *Slow 3G* in the devtools, then opening the profile page.

Reply:
okay, that makes sense - could you comment it to explain the function of the loader class? :) otherwise I'm sure I won't remember in three months time.